src/example/_version.py

```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this module."""
-__version__ = "0.0.1"
+__version__ = "0.1.0"
```

Comment:
Why is this file still here?

Reply:
Because #9 has not been merged in yet.  Once that happens, everything should look correct.